src/test/java/api/loans/DeclareClaimedReturnedItemAsMissingApiTests.java

```diff
@@ -104,11 +114,27 @@ public void cannotDeclareItemMissingWhenLoanIsNotFound() {
       .attemptDeclareClaimedReturnedItemAsMissing(
         new DeclareClaimedReturnedItemAsMissingRequestBuilder()
           .forLoan(notExistentLoanId)
-          .withComment("testing"));
+          .withComment(TESTING));
 
     assertThat(response.getStatusCode(), is(404));
   }
 
+  @Test
+  public void noteCreatedWhenDeclaredMissing() {
+    int initalNoteCount = notesClient.getAll().size();
+
+    claimItemReturnedFixture.claimItemReturned(new ClaimItemReturnedRequestBuilder()
+      .forLoan(loanId)
+      .withItemClaimedReturnedDate(DateTime.now()));
+
+    claimItemReturnedFixture.declareClaimedReturnedItemAsMissing(
+      new DeclareClaimedReturnedItemAsMissingRequestBuilder()
+        .forLoan(loanId)
+        .withComment(TESTING));
+
+    assertEquals(1, notesClient.getAll().size() - initalNoteCount);
```

Comment:
Why would there be other notes present during this test?

Could this instead check that the specific note that was intended to be created was present?